packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -417,14 +419,17 @@ test.describe('modal', () => {
                 expect(isModalVisible).toBe(true);
             });
 
-            test('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
+            test.skip('should NOT close the modal when the Escape key is pressed', async ({ mount, page }) => {
```

Comment:
expected to skip?

Reply:
Oh yes, sorry, forgot to add a comment in the PR for this. Will do that now.

This is a legitimate failure that needs to be looked at but was blocking my PR. Will write a ticket to address this issue.

Reply:
Update: DSW-2218 has been created to address this test.